diff --git a/dist/exports/StyleSheet/index.js b/dist/exports/StyleSheet/index.js
index 891994c511839216d523e68786daea85261539fb..41d86b9b47e9f2199839cda87c32ce375b8a3765 100644
--- a/dist/exports/StyleSheet/index.js
+++ b/dist/exports/StyleSheet/index.js
@@ -11,7 +11,7 @@ var _excluded = ["writingDirection"];
  */
 
 import { atomic, classic, inline } from './compiler';
-import { createSheet } from './dom';
+import { createSheet } from 'react-native-web-tailwind-compat';
 import { localizeStyle } from 'styleq/transform-localize-style';
 import { preprocess } from './preprocess';
 import { styleq } from 'styleq';
