diff --git a/dist/exports/StyleSheet/dom/createCSSStyleSheet.js b/dist/exports/StyleSheet/dom/createCSSStyleSheet.js
index 5d14a5f06c66ca41e4d1652c316b797bd4824eb1..e5ecfed752739a39544f3e67ee5086070e3d1260 100644
--- a/dist/exports/StyleSheet/dom/createCSSStyleSheet.js
+++ b/dist/exports/StyleSheet/dom/createCSSStyleSheet.js
@@ -12,6 +12,7 @@ import canUseDOM from '../../../modules/canUseDom';
 export default function createCSSStyleSheet(id, rootNode, textContent) {
   if (canUseDOM) {
     var root = rootNode != null ? rootNode : document;
+    console.log('RNW reading stylesheet')
     var element = root.getElementById(id);
     if (element == null) {
       element = document.createElement('style');
